src/api/termDuration.ts

```
@@ -0,0 +1,121 @@
+ * Convert an IS duration string to minutes.
+ *
+ * Verified format (real IS Mendelu sample, 2026-07): "10 minut".
+ * The hour-length rendering has NOT been observed on a real page yet, so the
+ * hour/combined/clock branches below are defensive. Anything unrecognised is
+ * reported via telemetry rather than guessed at — that way real-world formats
+ * surface as data instead of silently degrading to the 90-minute fallback.
+ *
+ * Returns null for IS empty sentinels, blanks, and unparseable input.
+ */
+export function parseDurationText(raw: string): number | null {
+    const text = (raw ?? '').replace(/ /g, ' ').trim();
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)